package.json

```diff
@@ -29,6 +29,9 @@
     "@mapbox/mapbox-sdk": "^0.15.3",
     "@material-ui/core": "^1.5.1",
     "@material-ui/icons": "^1.1.1",
+    "@mui/x-data-grid": "^7.6.1",
+    "@emotion/styled": "^11.11.5",
+    "@emotion/react": "^11.4.0",
```

Comment:
I used the x-data-grid react component because the Table react component in the legacy version of material-ui being used for this project does not support virtualized paging and I assumed live datasets could contain a larger number of segments and files than the demo dataset provided.  If this assumption was incorrect, I can update it to use the Table react component instead.